demos/pager/css/pager.css

```diff
@@ -127,3 +127,14 @@
  .demo-pager-page-size  label {
     margin-right: 2px;
  }
+
+ /***********************************************
+ * Styles for pager previous and next page symbol
+ */
+ .demo-pager-previous-character::before {
+     content: "<";
+ }
+
+ .demo-pager-next-character::after {
+     content: ">";
+ }
```

Comment:
please add an empty new line character at the end of the file.

Reply:
Done

Reply:
Done